maas-api.yaml

```diff
@@ -1110,1966 +1151,1914 @@ paths:
+          type: array
+          items:
+            type: array
+            items:
+              type: object
+              properties:
+                occupancy:
+                  type: string
+                  description: How full this part of the vehicle is. See https://developers.google.com/transit/gtfs-realtime/gtfs-realtime-proto.
+                  enum:
+                    - EMPTY
+                    - MANY_SEATS_AVAILABLE
+                    - FEW_SEATS_AVAILABLE
+                    - STANDING_ROOM_ONLY
+                    - CRUSHED_STANDING_ROOM_ONLY
+                    - FULL
+                    - NOT_ACCEPTING_PASSENGERS
+                wifi:
+                  type: boolean
+                  description: Whether this part of the vehicle has Wi-Fi service.
+                airConditioned:
+                  type: boolean
+                  description: Whether this part of the vehicle is equiped with air conditioned.
+                wheelchairAccessible:
+                  type: boolean
+                  description: Whether this part of the vehicle is wheel chair capable.
+                wheelchairSeats:
+                  type: integer
+                  description: Indicates the number of wheel chairs that can be accommodated on this vehicle.
+                model:
+                  type: string
+                  description: Indicates the type of vehicle performing this trip. Please check agency documentation to determine how this field is composed.
+      required:
+        - id
+        - lastUpdate
+        - location
+
+    Region:
+      properties:
+        id:
+          type: string
+          description: Identifier for the region
+        cities:
+          type: array
+          description: List of bigger cities covered by this region. Note that this can be empty.
+          minItems: 0
+          items:
+            type: object
+            properties:
+              lat:
+                type: number
+              lng:
+                type: number
+              title:
+                type: string
+              timezone:
+                type: string
+            required:
+              - lat
+              - lng
+              - title
+              - timezone
+
+        polygon:
+          type: string
+          description:
+            Encoded polygon of coverage area for the region, using
+            Google's Encoded Polyline Algorithm. See
+            https://developers.google.com/maps/documentation/utilities/polylinealgorithm.
+        modes:
+          type: array
+          description:
+            List of mode identifiers that are available in this region (but not necessarily everywhere in that region).
+            Look at `modes` array in `/regions` to get more details on each mode. These are also the modes that can be
+             used as input to any routing endpoint. See
+             https:
```

Comment:
are we going to continue using `SegmentTemplate`?

Reply:
Would be good to clean that up. I don't mind if `/routing/grouped` doesn't make it into this spec.